packages/bottender/src/cli/shared/getWebhookFromNgrok.ts

```diff
@@ -1,12 +1,62 @@
 import axios from 'axios';
-import get from 'lodash/get';
 
-export default async function getWebhookFromNgrok(ngrokPort: string) {
+type Tunnel = {
+  name: string;
+  uri: string;
+  public_url: string;
+  proto: string;
+  config: {
+    addr: string;
+    inspect: boolean;
+  };
+  metrics: {
+    conns: {
+      count: number;
+      gauge: number;
+      rate1: number;
+      rate5: number;
+      rate15: number;
+      p50: number;
+      p90: number;
+      p95: number;
+      p99: number;
+    };
+    http: {
+      count: number;
+      rate1: number;
+      rate5: number;
+      rate15: number;
+      p50: number;
+      p90: number;
+      p95: number;
+      p99: number;
+    };
+  };
+};
+
+export default async function getWebhookFromNgrok(
+  ngrokPort = '4040'
+): Promise<string | never> {
+  // ngork API reference: https://ngrok.com/docs#client-api
   const ngrokAxios = axios.create({
     baseURL: `http://localhost:${ngrokPort}`,
   });
 
-  // TODO: use ngrok Client API? - https://ngrok.com/docs#client-api
-  const res = await ngrokAxios.get('/api/tunnels');
-  return get(res, 'data.tunnels[1].public_url'); // tunnels[1] return `https` protocol
+  const { data }: { data: { tunnels: Tunnel[] } } = await ngrokAxios.get(
```

Comment:
Write this with generic type is better:

```js
const { data } = await ngrokAxios.get<{ tunnels: Tunnel[] }>(
  '/api/tunnels'
);
```

https://github.com/axios/axios/blob/master/index.d.ts#L133